package.json

```diff
@@ -49,6 +49,7 @@
     "humanize-string": "^3.0.0",
     "maplibre-gl": "^3.1.0",
     "pmtiles": "^2.10.0-beta.0",
+    "read-excel-file": "^5.6.1",
```

Comment:
There's https://www.npmjs.com/package/xlsx and some other heavier options, but this one seems perfectly adequate for us